module.txt

```diff
@@ -7,7 +7,7 @@
     "serverSideOnly": false,
     "dependencies": [
         { "id": "CoreAdvancedAssets", "minVersion": "1.1.0" },
-        { "id": "Health", "minVersion": "1.0.1", "optional": true, "because": "tests"},
+        { "id": "Health", "minVersion": "1.0.1", "optional": true},
```

Comment:
I think this was me. I don't blame you for taking it out as I don't think the `because` value is part of the schema gestalt knows about, but we were trying to figure out some way of distinguishing dependencies that are there for gameplay vs those that are there only for the tests.

Reply:
I was wondering what that `because` thing meant when I saw it for the first time the other day! I had sort of figured this was it